templates/opa/hello-rego.tmpl

```diff
@@ -0,0 +1,20 @@
+package policies.hello
```

Comment:
could this include a verb like we have in other templates?
having one feels more like an action that needs a decision - then the visible and enabled make more sense?

```suggestion
package policies.GET.hello
```

Reply:
Without context, I do not think this should be in the init template